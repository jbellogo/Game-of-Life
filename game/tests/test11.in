new 4
init 
0 0
2 2
1 1
3 3
-1 -1
print
