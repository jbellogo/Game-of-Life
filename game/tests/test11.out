X___
_X__
__X_
___X
